openjdk/playlist.xml

```diff
@@ -1757,4 +1757,215 @@
 			<group>openjdk</group>
 		</groups>
 	</test>
+	<test>
+		<testCaseName>dragonwell8_feature_jdk0</testCaseName>
+		<command>$(JAVA_COMMAND) -Xmx512m -jar $(Q)$(TEST_RESROOT)$(D)jtreg$(D)lib$(D)jtreg.jar$(Q) \
+			$(JTREG_BASIC_OPTIONS) -vmoptions:$(Q)-Xmx512m $(JVM_OPTIONS)$(Q) \
+			-w $(Q)$(REPORTDIR)$(D)work$(Q) \
+			-r $(Q)$(TEST_RESROOT)$(D)report$(Q) \
+			-jdk:$(Q)$(TEST_JDK_HOME)$(Q) \
+			-exclude:$(Q)$(JTREG_JDK_TEST_DIR)$(D)ProblemList.txt$(Q) \
+			-exclude:$(Q)$(TEST_RESROOT)$(D)$(PROBLEM_LIST_FILE)$(Q) \
+			$(Q)$(JTREG_JDK_TEST_DIR)/com/alibaba/wisp2$(Q); \
+			$(TEST_STATUS)
+		</command>
+		<versions>
+			<version>8</version>
+			<version>11</version>
+		</versions>
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>openjdk</group>
+		</groups>
```

Comment:
```suggestion
		</groups>
		<vendor>alibaba</vendor>
```

Reply:
Since each of these new `<test>` blocks are specific to alibaba dragonwell, please add `<vendor>` tags to all newly added `<test>` blocks so they will not be executed against temurin or others.